targets/xbmc

```diff
@@ -1,35 +0,0 @@
-#!/bin/sh -e
```

Comment:
I don't think we can outright delete this target; we need to provide a migration path.

One quick-and-dirty solution would be to make xbmc a minimal target like [x11](https://github.com/dnschneid/crouton/blob/90ad47ff22e148db41ba627992ee5cc5d0c14175/targets/x11), with an unconditional `REQUIRES=kodi`, and add `PROVIDES='xbmc'` to the kodi target (like in [xorg](https://github.com/dnschneid/crouton/blob/90ad47ff22e148db41ba627992ee5cc5d0c14175/targets/xorg)).  Perhaps even leave DESCRIPTION blank and update installer/main.sh to avoid printing out any targets that have a blank DESCRIPTION for `-t list`.

The best solution would be to make xbmc a symlink to kodi, and then add some smarts to installer/main.sh to interpret a symlinked target as if it were the above.  This would be a little tricky (but not too hard) because the script would have to automatically populate PROVIDES by detecting any target symlinks that point to the selected target.